code/game/objects/items/weapons/grenades/chem_grenade.dm

```diff
@@ -278,7 +278,7 @@
 	beakers += B2
 
 
-/obj/item/weapon/grenade/chem_grenade/incendiary

```

Comment:
correct me if i'm wrong but won't this give EVERY chem grenade the properties of an incendiary grenade? Since this isn't a sub type?